internal/k3d/terraform.go

```diff
@@ -53,3 +58,50 @@ type GithubTerraformEnvs struct {
 	AwsAccessKeyId        string
 	AwsSecretAccessKey    string
 }
+
+func TerraformPrep(config *K3dConfig) error {
+
+	path := config.GitopsDir + "/terraform"
+	log.Info().Msgf("Repo is %s", path)
+	err := filepath.Walk(path, detokenizeterraform(path, config))
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func detokenizeterraform(path string, config *K3dConfig) filepath.WalkFunc {
+	return filepath.WalkFunc(func(path string, fi os.FileInfo, err error) error {
+
+		if fi.IsDir() {
+			return nil
+		}
+
+		matched, _ := filepath.Match("*", fi.Name())
```

Comment:
Not sure where the comment went but wouldn't this match every single file?

Reply:
ohh it was even present in the function for other clouds